{
  "cve": "CVE-2020-26159",
  "mitre": {
    "cpes": [],
    "created": "2020-09-30T12:41:13+00:00",
    "description": "DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Further investigation showed that it was not a security issue. Notes: none",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "mitre_repo_path": "cves/2020/26xxx/CVE-2020-26159.json",
    "references": [],
    "title": null,
    "updated": "2021-02-02T20:16:32+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "nvd": {
    "cpes": [],
    "created": "2020-09-30T21:15:13.357000+00:00",
    "description": "Rejected reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Further investigation showed that it was not a security issue. Notes: none",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {},
      "cvssV4_0": {}
    },
    "nvd_repo_path": "2020/CVE-2020-26159.json",
    "references": [],
    "title": null,
    "updated": "2023-11-07T03:20:30.767000+00:00",
    "vendors": [],
    "weaknesses": []
  },
  "opencve": {
    "changes": [],
    "cpes": {
      "data": [],
      "providers": []
    },
    "created": {
      "data": "2020-09-15T00:00:00+00:00",
      "provider": "redhat"
    },
    "description": {
      "data": "DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Further investigation showed that it was not a security issue. Notes: none",
      "provider": "mitre"
    },
    "metrics": {
      "cvssV2_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_0": {
        "data": {},
        "provider": null
      },
      "cvssV3_1": {
        "data": {
          "score": 8.6,
          "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:H"
        },
        "provider": "redhat"
      },
      "cvssV4_0": {
        "data": {},
        "provider": null
      },
      "kev": {
        "data": {},
        "provider": null
      },
      "ssvc": {
        "data": {},
        "provider": null
      },
      "threat_severity": {
        "data": "Moderate",
        "provider": "redhat"
      }
    },
    "references": {
      "data": [
        "https://github.com/kkos/oniguruma/commit/cbe9f8bd9cfc6c3c87a60fbae58fa1a85db59df0",
        "https://github.com/kkos/oniguruma/issues/207",
        "https://nvd.nist.gov/vuln/detail/CVE-2020-26159",
        "https://www.cve.org/CVERecord?id=CVE-2020-26159"
      ],
      "providers": [
        "redhat"
      ]
    },
    "title": {
      "data": "oniguruma: Buffer overflow in concat_opt_exact_str could result in DoS",
      "provider": "nvd"
    },
    "updated": {
      "data": "2023-11-07T03:20:30.767000+00:00",
      "provider": "nvd"
    },
    "vendors": {
      "data": [],
      "providers": []
    },
    "weaknesses": {
      "data": [
        "CWE-193",
        "CWE-787"
      ],
      "providers": [
        "redhat"
      ]
    }
  },
  "redhat": {
    "cpes": [],
    "created": "2020-09-15T00:00:00+00:00",
    "description": "This CVE ID has been rejected or withdrawn by its CVE Numbering Authority for the following reason: DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Further investigation showed that it was not a security issue. Notes: none",
    "metrics": {
      "cvssV2_0": {},
      "cvssV3_0": {},
      "cvssV3_1": {
        "score": 8.6,
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:L/A:H"
      },
      "threat_severity": "Moderate"
    },
    "redhat_repo_path": "2020/CVE-2020-26159.json",
    "references": [
      "https://github.com/kkos/oniguruma/commit/cbe9f8bd9cfc6c3c87a60fbae58fa1a85db59df0",
      "https://github.com/kkos/oniguruma/issues/207",
      "https://nvd.nist.gov/vuln/detail/CVE-2020-26159",
      "https://www.cve.org/CVERecord?id=CVE-2020-26159"
    ],
    "title": "oniguruma: Buffer overflow in concat_opt_exact_str could result in DoS",
    "updated": "2020-09-15T00:00:00+00:00",
    "vendors": [],
    "weaknesses": [
      "CWE-193",
      "CWE-787"
    ]
  }
}